src/org.xtuml.bp.core/src/org/xtuml/bp/core/util/UIUtil.java

```diff
@@ -806,4 +806,29 @@ public void inputChanged(Viewer viewer, Object oldInput, Object newInput) {
 		}
 
 	}
+
+	public static boolean isDarkTheme() {
+		// get the current workbench theme, set defaults accordingly
+		// NOTE, we could import the swt css libraries but they are not
+		// considered API and access is discouraged. Instead of adding
+		// the dependencies we just use reflection, worse case is we
+		// revert to light mode

```

Comment:
How are other plugins accessing the theme? Surely we are not the only one with custom elements that we want to follow the current theme.